lib/eu_central_bank/version.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+class EuCentralBank < Money::Bank::VariableExchange
```

Comment:
Ops, the test failure is legit. It should not inherit from `Money::Bank::VariableExchange` 😊

Reply:
ok!